morpheus/_lib/src/objects/tensor_object.cpp

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Just so I'm clear, this is all code that was basically pulled in from SRF, right?

Reply:
Yes, with the only changes being the names of the files, and the namespace.
In the case of `type_util.hpp` there were two structs and a method defined in the global namespace that were removed because they weren't used by morpheus and clashed with those same symbols defined in neo.